packages/cozy-sharing/src/components/Recipient.jsx

```diff
@@ -135,11 +124,7 @@ export class Status extends Component {
         )}
         {shouldShowMenu && (
           <MenuAwareMobile
-            label={
-              status === 'ready' && type
-                ? t(`Share.type.${type}`)
-                : t(`Share.status.${status}`)
-            }
+            text={t(`Share.type.${type}`)}
```

Comment:
IIRC we showed the status when accepted, and the permission type otherwise. Now we always show the type here, and the status is displayed in another part of the UI.